web/styles/left_sidebar.css

```diff
@@ -972,6 +972,22 @@ li.top_left_scheduled_messages {
            definition. */
         height: auto;
     }
+
+    .fa {
+        color: var(--color-left-sidebar-navigation-icon);
+    }
+
+    .fa:hover {
+        color: var(--color-text-message-header);
+    }
+}
+
+.bottom_left_row .fa-align-right {
+    display: none;
+}
+
+.bottom_left_row:hover .fa-align-right {
+    display: inline-block;
```

Comment:
We only want to apply css for `.stream-markers-and-controls .fa-align-right`. Let's directly use it. Also, you can use `&:hover` to shorten your css.

Reply:
If we use `.stream-markers-and-controls .fa-align-right`, the interleaved button will only appear when the user hovers over the unread counter, and it will not appear when the stream is hovered over.

**Stream hover:**
![image](https://github.com/zulip/zulip/assets/73663475/1ac922b6-faaf-4a52-9c8f-ae6a6d1cb3c4)

**Unread counter hover:**
![image](https://github.com/zulip/zulip/assets/73663475/91b0e9a4-93f0-4641-b7ac-85c064064f77)

```css

    .fa-align-right {
        color: var(--color-left-sidebar-navigation-icon);
        display: none;
    }
    
    &:hover {
        color: var(--color-text-message-header);
        .fa-align-right {
            display: inline-block;
        }
    }
}

```